doc/howtos/software-update-server.rst

```diff
@@ -158,31 +156,31 @@ customized image, by:
    syntax. The other identifiers are under the developer's control.
 
 
-This will add, to the set of images that are buildable, one referred to as
-``ostro-image-swupd-pre_installed_content``, which contains, besides the
-os-core bundle, also the ``sudo_bundle`` bundle from Step 1 and whatever
+This will add a new image option called
+``ostro-image-swupd-pre_installed_content``
+to the set of images that are buildable. This new image
+contains the
+``os-core bundle``,  the ``sudo_bundle`` bundle from Step 1, and whatever
 else might have been added.
 
-The main difference, compared to the approach taken in Step 0 is that
-this approach allows for removing the content of the bundle without having to
+Compared to the approach taken in Step 0, 
+this approach allows you to remove the content of the bundle without having to
 create an update (and thus a new release).
 
 .. note::
-   Each bundle defined generates a non-negligible amount of load, when
-   building images and SW Update repositories.
-
-   It is therefore recommended to keep enabled only those bundles that
+   Each defined bundle generates an additional workload when
+   building images and SW Update repositories so we recommend
+   you enable only those bundles that
    are effectively useful for the specific use-case targeted.
 
-   In case one wants to modify the set of predefined bundles that come
-   from the defaults of Ostro OS, it is possible to override the variable
-   ``SWUPD_BUNDLES``.
-
-   Doing so will discard anything that was part of the defaults, therefore
-   if any of the pre-defined bundles is still needed, it must be listed
-   again, explicitly.
+You can override the set of predefined default bundles that come
+from Ostro OS, by editing the
+``SWUPD_BUNDLES`` variable.  Doing so will discard anything that was 
+part of the Ostro OS defaults;
+if any of the pre-defined bundles are still needed, they must be 
+explicitly listed again.
```

Comment:
I'd state "products should override..." as no real product would ever use our bundles :-)